components/EvmSend.tsx

```diff
@@ -49,6 +49,7 @@ const EvmSend = ({ functionName, buildArguments, feeKey }: EvmSendProps) => {
   useEffect(() => {
     if (isLoading) {
       setResponsePreface("Loading...");
+      setResponse("");
```

Comment:
drive-by bug fix: this used to show the old response while loading the new one

Reply:
thanks for the fix